gulpfile.js

```diff
@@ -27,6 +27,7 @@ plugins.uglify = require('gulp-uglify');
 const paths = {
   src: 'app/assets/',
   dist: 'app/static/',
+  templates: 'app/templates/',
```

Comment:
you might have a merge conflict here. I believe Alex removed this line

Reply:
I removed the extra line and synced with main in order for this PR to get merged today